client.cc

```diff
@@ -258,6 +260,17 @@ Packet Client::LTOCompile (char const *const *argv, size_t argc)
   return MaybeRequest (Detail::RC_LTO_COMPILE);
 }
 
+// LTO-RESPONSE $message
+Packet LTOResponse (std::vector<std::string> &words)
+{
+  if (words[0] == u8"LTO-RESPONSE") {
```

Comment:
don't need the extraneous bracing (and note they're indented incorrectly if we did)

Reply:
ah yes, old habits..